src/fr/composants/fil-dariane/case-dusage.md

```diff
@@ -0,0 +1,14 @@
+---
+title: fil-dariane - Composants
```

Comment:
Nick recommended using `Chemin de navigation` as the french translation for `Breadcrumbs` instead of `Fil d\'Ariane`.